test/shell.test.ts

```diff
@@ -49,11 +49,24 @@ describe("commands/shell", function () {
     );
   });
 
-  test("Shell throws without network", async function () {
```

Comment:
This is kind of a nit.  The term network seems ambiguous, and we are testing the `--chain` flag.

Reply:
Nits are good, this is a good one.